specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2018-11-20/examples/createOrUpdateGuestConfigurationAssignment.json

```diff
@@ -20,9 +20,8 @@
             }
           ],
           "configurationSetting": {
-            "rebootIfNeeded": "False",
+            "rebootIfNeeded": false,
             "actionAfterReboot": "ContinueConfiguration",
-            "configurationModeFrequencyMins": 15,
```

Comment:
Why do we have to remove this in the example? This is supported by the schema

Reply:
Added back in. I was trying to fix the validation error.